src/main/java/homework_4/singleton/Singleton.java

```diff
@@ -0,0 +1,15 @@
+package homework_4.singleton;
+
+public class Singleton {
+
+    private Singleton() {
+    }
+
+    private static class SingletonHolder {
+        public static final Singleton INSTANCE_HOLDER = new Singleton();
+    }
+
+    public static Singleton getInstance() {
+        return SingletonHolder.INSTANCE_HOLDER;
+    }
```

Comment:
Opt: why do you need static class, why not to have just static variable?

Reply:
![image](https://user-images.githubusercontent.com/86798105/134068094-8c44d744-5069-42f9-9702-7f6641a97d7c.png)

Reply:
Ok, agree